homeworks/olya.kosenko_olyabot/homework_2/Task_1.js

```diff
@@ -0,0 +1,33 @@
+function PYRAMID(n1) {
+    console.log(`PYRAMID with ${n1} rows:`);
+    const str1 = '#';
+    const str2 = ' ';
+    for (let i = 0; i < n1; i++) {
+        console.log(str2.repeat(n1 - i) + str1.repeat(i * 2 + 4));
+    }
+}
+
+function check(n) {
```

Comment:
this function with name check should return true/false, but shouldn't execute PYRAMID function or make any  output